scripts/update_vertexai_responses.sh

```diff
@@ -17,6 +17,10 @@
 # This script replaces mock response files for Vertex AI unit tests with a fresh
 # clone of the shared repository of Vertex AI test data.
 
+RESPONSES_VERSION='v1.*' # the major version of mock responses to use
+REPO="https://github.com/FirebaseExtended/vertexai-sdk-test-data.git"
+TAG=$(git -c 'versionsort.suffix=-' ls-remote --tags --sort='v:refname' "$REPO" | grep "$RESPONSES_VERSION" | tail -n1 | awk -F'/' '{print $NF}')
```

Comment:
Add a comment here to explain what the command does

Reply:
Does the output include any `/` ? it's not clear either why the awk is needed here.